embeddings/pipeline/hugging_face_pair_classification.py

```diff
@@ -27,14 +30,16 @@ def __init__(
         input_columns_names_pair: Tuple[str, str],
         target_column_name: str,
         output_path: T_path,
+        sample_missing_splits: Tuple[float, float] = (0.1, 0.1),
```

Comment:
Same as previously: In my view sampling in pipelines should be optional instead of forcing sampling.